en/docs/learn/analytics/customizing-analytics-dashboards.md

```diff
@@ -0,0 +1,27 @@
+#Customizing Analytics Dashboards
+
+API Manager analytics provides three types of dashboards which are APIM Admin, APIM Developer Portal and APIM Publisher.
+
+The users will not be allowed to modify the default dashboards, i.e either to modify the layout of the widget or add custom widgets to a particular dashboard. If it is required to modify the default dashboard, a copy of the dashboard should be created and the modification should be applied to that copy as described below.
+
+1.  Click on the button at the bottom right corner of the dashboard that needs to be customized as shown below.
+
+    ![API Manager default dashboards](../../assets/img/learn/apim-analytics-default-dashboards.png)
+    
+2.  A dropdown menu will appear as shown below and click the `Duplicate` button from the dropdown.
+    
+    ![APIM Analytics dashboard dropdown](../../assets/img/learn/apim-analytics-dashboard-dropdown.png)
+    
+3.  A form will be displayed as shown below. Add a valid `Name` and `URL` for the dashboard according to the preference and click the `ok` button.
+
+    ![Dashboard duplication form](../../assets/img/learn/apim-analytics-dashboard-duplication-form.png)
+    
+4.  Now a copy of the dashboard is created with the provided `Name` as shown below.
+    
+    ![Duplicated dashboard](../../assets/img/learn/apim-analytics-duplicated-dashboard.png)
+    
+5.  Again click the dropdown button at the bottom right corner of the created dashboard and click the `design` button as shown below.
```

Comment:
let's remove the word 'Again'

Reply:
Resolved with ba9b2a4d35f938794a98f8eb2d67f37c0f9f911a